### PR TITLE
[DOCS] Link to Elastic Rerank model landing page

### PR DESCRIPTION
**Pull request by leemthompo**
Page is live so can link to it now

### CODE REVIEW

File: docs/reference/search/retriever.asciidoc [hunk 1]
```diff
@@ -442,7 +442,7 @@ If the child retriever already specifies any filters, then this top-level filter
 [[text-similarity-reranker-retriever-example-elastic-rerank]]
 ==== Example: Elastic Rerank
 
-This examples demonstrates how to deploy the Elastic Rerank model and use it to re-rank search results using the `text_similarity_reranker` retriever.
+This examples demonstrates how to deploy the {ml-docs}/ml-nlp-rerank.html[Elastic Rerank] model and use it to re-rank search results using the `text_similarity_reranker` retriever.
```
Comment (kosabogi) [inline docs/reference/search/retriever.asciidoc:445]:
```suggestion
This example demonstrates how to deploy the {ml-docs}/ml-nlp-rerank.html[Elastic Rerank] model and use it to re-rank search results using the `text_similarity_reranker` retriever.
```

**Review by kosabogi — COMMENTED** [review]
Just a small typo :)

**Review by kosabogi — APPROVED** [review]
🎉

File: docs/reference/inference/service-elasticsearch.asciidoc [hunk 2]
```diff
@@ -153,7 +153,7 @@ For further details, refer to the {ml-docs}/ml-nlp-elser.html[ELSER model docume
 [[inference-example-elastic-reranker]]
 ==== Elastic Rerank via the `elasticsearch` service
 
-The following example shows how to create an {infer} endpoint called `my-elastic-rerank` to perform a `rerank` task type using the built-in Elastic Rerank cross-encoder model.
+The following example shows how to create an {infer} endpoint called `my-elastic-rerank` to perform a `rerank` task type using the built-in {ml-docs}/ml-nlp-rerank.html[Elastic Rerank] cross-encoder model.
```
Comment (leemthompo) [inline docs/reference/inference/service-elasticsearch.asciidoc:156]:
```suggestion
The following example shows how to create an {infer} endpoint called `my-elastic-rerank` to perform a `rerank` task type using the built-in {ml-docs}/ml-nlp-rerank.html[Elastic Rerank] cross-encoder model.

Refer to this https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/12-semantic-reranking-elastic-rerank.ipynb[Python notebook] for an end-to-end example using Elastic Rerank.
```

File: docs/reference/search/retriever.asciidoc [hunk 3]
```diff
@@ -442,7 +442,7 @@ If the child retriever already specifies any filters, then this top-level filter
 [[text-similarity-reranker-retriever-example-elastic-rerank]]
 ==== Example: Elastic Rerank
 
-This examples demonstrates how to deploy the Elastic Rerank model and use it to re-rank search results using the `text_similarity_reranker` retriever.
+This example demonstrates how to deploy the {ml-docs}/ml-nlp-rerank.html[Elastic Rerank] model and use it to re-rank search results using the `text_similarity_reranker` retriever.
```
Comment (leemthompo) [inline docs/reference/search/retriever.asciidoc:445]:
```suggestion
[TIP]
====
Refer to this https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/12-semantic-reranking-elastic-rerank.ipynb[Python notebook] for an end-to-end example using Elastic Rerank.
====

This example demonstrates how to deploy the {ml-docs}/ml-nlp-rerank.html[Elastic Rerank] model and use it to re-rank search results using the `text_similarity_reranker` retriever.
```

**Pull request merged**